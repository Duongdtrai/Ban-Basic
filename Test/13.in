4
7
9261 90170 50814 28296 37785 64488 33614 
5
9985 63387 72331 75622 99864 
9
91710 92702 59811 10171 38480 33006 3433 65240 51017 
9
5611 77885 87327 89236 34833 83382 1401 74315 17323 
