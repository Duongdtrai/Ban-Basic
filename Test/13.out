113044480548
157931881854
523142949300
147824400752
